### DISCUSSION TITLE
Is it possible to add more than 1 Reolink camera?

### DISCUSSION THREAD

**Discussion by PuzzleDog** [fwestenberg/reolink_dev#487] [General]
I have several supported Reolink cameras, and I was successful in configuring one camera.  It appears in Home Assistant just like it should.

Is it possible to add more than 1 Reolink camera to 'reolink_dev'?  If so, how?  I haven't been able to find a configuration file anywhere that contains the IP address of my first camera.

**Comment by hobbyboy** [comment]
Just re-add the itegration, like you have not done it yet.  A second will show up in the existing integration.

**Comment by PuzzleDog** [comment]
That worked perfectly!  Thank you very much for answering my question.  Greatly appreciated.
